functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,35 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals isPalindrome*/
+
+describe("Checking if word is palindrome: ", function () {
+    it("word 'tenet' is a palindrome", function () {
+        var palindrome = isPalindrome("tenet");
+        expect(palindrome).toBeTruthy();
+    });
+    it("word dog isn't a palindrome", function () {
+        var palindrome = isPalindrome("dog");
+        expect(palindrome).toBeFalsy();
+    });
+    it("if after palidromem is spacia 'kayak ', it is palidrom too", function () {
+        var palindrome = isPalindrome("kayak ");
+        expect(palindrome).toBeTruthy();
+    });
+    it("if before palidromem is spacia ' kayak', it is palidrom too", function () {
+        var palindrome = isPalindrome(" kayak");
+        expect(palindrome).toBeTruthy();
+    });
+    it("if between two palindromes is spacia 'kayak kayak', it is a palindrome too", function () {
```

Comment:
Description is incorrect:
"if between two palindromes is spacia, it is a palindrome too"
are you sure about it?
What about:
'kayak eye'
It consists of two palindromes separates with space. But is the whole content a palindrome?
It would be good to change this description to be more details and maybe also add another test for such example 'kayak eye'